test/services/return-requirements/submit-returns-cycle.service.test.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
+
+// Thing under test
+const SubmitReturnsCycleService = require('../../../app/services/return-requirements/submit-returns-cycle.service.js')
+
+describe('Submit Returns Cycle service', () => {
+  let session
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+
+    session = await SessionHelper.add({
+      data: {
+        licence: {
+          id: '8b7f78ba-f3ad-4cb6-a058-78abc4d1383d',
+          currentVersionStartDate: '2023-01-01T00:00:00.000Z',
+          endDate: null,
+          licenceRef: '01/ABC',
+          licenceHolder: 'Turbo Kid',
+          startDate: '2022-04-01T00:00:00.000Z'
+        },
+        journey: 'returns-required'
+      }
+    })
+  })
+
+  describe('when called', () => {
+    describe('with a valid payload', () => {
+      const payload = {
```

Comment:
Since you are using payload twice I would put payload as a let and then put them both in beforeEach blocks instead of const.